services/core/shared/core/compat/sdk_v4/schema/transactions.js

```diff
@@ -0,0 +1,45 @@
+
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+module.exports = {
+	primaryKey: 'id',
+	schema: {
+		id: { type: 'string' },
+		type: { type: 'integer' },
+		senderId: { type: 'string' },
+		recipientId: { type: 'string' },
+		amount: { type: 'integer' },
```

Comment:
value of `amount` is of type `BigInt`. (While writing amount to `mysql`, it throws an warning: `Error: ER_WARN_DATA_OUT_OF_RANGE: Out of range value for column 'amount' `)

Update to:
```
amount: { type: 'bigInteger' },
```